collectors/0/flume.py

```diff
@@ -92,13 +92,13 @@ def main(argv):
     err("This collector requires the `json' Python module.")
     return 1
 
-  def printmetric(metric, value, **tags):
+  def printmetric(component, metric, value, **tags):
     if tags:
       tags = " " + " ".join("%s=%s" % (name, value)
                             for name, value in tags.iteritems())
     else:
       tags = ""
-    print ("flume.%s %d %s %s" % (metric, ts, value, tags))
+    print ("flume.%s.%s %d %s %s" % (component, metric, ts, value, tags))
```

Comment:
seems like it's getting split on line 109 and then re-joined. any reason to not just pass the metric value on 109 and keep it as key?